### ISSUE TITLE
Subfolder Reverse Proxy Support

### ISSUE THREAD

**Issue by SubrataM3** [pydio/cells#386]
Is there any plan to introduce support for subfolder reverse proxy? It used to work fine on the PHP version but it is breaking badly on the cells.

**Comment by sirlordt** [comment]
Yes please!!!. Use with sub domain is a security issue. The hackers always look first in your dns zones looking by services and hosts like mypydio,mydomain.com is very easy spot and launch attack from this way to specific targets. Using sub folders you can hide your web apps from external eyes. like https://mydomain.com/my/secret/path. 

Is not 100% secure but is more difficult trace the url.
